Gruntfile.js

```diff
@@ -339,7 +339,7 @@ module.exports = function(grunt) {
       mozrule: {
         files: {"<%= config.buildFile %>" : "<%= config.buildFile %>"},
         options: {
-          wrapper: ["<%= config.prefix %>", "}\n"]
+          wrapper: ["<%= config.prefix %>", ""]
```

Comment:
What was this `}\n` for?

Reply:
That used to close the main `@-moz-document`.

Reply:
Ah, right so because the closing bracket is now preserved, we drop it here.